### DISCUSSION TITLE
modding tab on profile overlay doesn't work

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#17868] [Q&A] [answered]
https://user-images.githubusercontent.com/22381687/163815262-3a2aacb9-fdb5-4a74-b991-faca8fd0973c.mp4

**Comment by bdach** [comment, marked as answer]
because it's not implemented yet

**Reply by respektive** [reply]
I see, thanks for the info. I was searching for modding related issues but couldn't find anything for the profile tab. Since the button is there I thought this might have been regressed in some recent update.
